scripts/init-submodules-no-riscv-tools-nolog.sh

```diff
@@ -81,3 +81,5 @@ echo "# line auto-generated by init-submodules-no-riscv-tools.sh" >> env.sh
 echo '__DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]:-${(%):-%x}}")")"' >> env.sh
 echo "PATH=\$__DIR/bin:\$PATH" >> env.sh
 echo "PATH=\$__DIR/software/firemarshal:\$PATH" >> env.sh
+echo "-Dsbt.sourcemode=true" > .sbtopts
+echo "-Dsbt.workspace=$CHIPYARD_DIR/tools"  >> .sbtopts
```

Comment:
You should also update the `.gitignore` to not track the file.

Reply:
Thanks and the files are changed accordingly.